src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.Reflection.cs

```diff
@@ -22,10 +21,6 @@ internal partial class XamlFileGenerator
 		private Func<INamedTypeSymbol, string[]>? _findLocalizableDeclaredProperties;
 		private XClassName? _xClassName;
 		private string[]? _clrNamespaces;
-		private readonly static Func<INamedTypeSymbol, IPropertySymbol?> _findContentProperty;
-		private readonly static Func<INamedTypeSymbol, string, bool> _isAttachedProperty;
-		private readonly static Func<INamedTypeSymbol, string, INamedTypeSymbol> _getAttachedPropertyType;
-		private readonly static Func<INamedTypeSymbol, bool> _isTypeImplemented;
```

Comment:
We should be at least memoizing for the current compilation, this is queried very frequently and (used to) have a significant impact on generation performance.

Reply:
@jeromelaban Yup, we almost commented in the same time!

I noticed this PR indeed degrades performance. So I'll move to non-static memoization in RoslynMetadataHelper.